public/requests.js

```diff
@@ -0,0 +1,109 @@
+import md5_encode from './MD5.js'
+
+/**
+ *  добавление eventListener на кнопку поиска трека
+ */
+const search_input = document.querySelector('.header__search');
+search_input.addEventListener('keydown', function(e) {
+    if (e.keyCode === 13) {
+        send_request('GET','track.search',`&track=${search_input.value}`,search_input.value).then(
+            res => get_content('search',res.results.trackmatches.track)
+        )
+    }
+});
+
+/**
+ *  добавление eventListener на кнопку любимых треков
+ */
+const fav_button = document.querySelector('.fav');
+fav_button.addEventListener('click', function() {
+    send_request('GET','user.getlovedtracks','&user=Foxyb0y').then(
+        res => get_content('favorites',res.lovedtracks.track)
+    )
+});
+
+/**
+ *
+ * @param method метод Апи (GET или POST)
+ * @param request_method параметр запроса - метод
+ * @param require_param параметр определяет какой GET-запрос отправить
+ * @param track название трека
+ * @param artist имя артиста
+ * @returns {Promise} Выполненный промис
+ */
+async function send_request(method,request_method,require_param='',track= '',artist='') {
```

Comment:
Не очень нравится именование переменных.
Я бы сделал так
```suggestion
async function send_request(method, params) {
```

params это объект с полями
method, param, track и artist